bika/lims/browser/analysisrequest/templates/analysisrequest_view.pt

```diff
@@ -53,6 +74,20 @@
         </tal:customfields>
     </div>
 
+    <!-- Refactored Remarks Widget
+         https://github.com/senaite/senaite.core/pull/920 -->
+    <div id="remarks-widget">
+      <tal:remarks tal:condition="python:hasattr(context.aq_inner, 'schema') and 'Remarks' in context.schema"
```

Comment:
I'd like to have a header for remarks, as we have for the rest of sections. A good candidate could be https://iconmonstr.com/speech-bubble-comments-thin-svg/

Also, I would add the condition to the div that wraps the remarks. Even better, we know that AR will always have remarks, so there is no need of that condition. Neither the need of the tal element, the tal expressions can be added to the div directly, like follows:

```html
<div id="remarks-widget" tal:define="checkPermission nocall: context/portal_membership/checkPermission;
                                     mode python:'edit' if checkPermission('Modify portal content', context) else 'view';
                                     field python:context.Schema()['Remarks'];
                                     errors python:{};">
  <h3>
    <img i18n:attributes="title" title="Remarks" src="++resource++bika.lims.images/remarks.png"/>
    <span i18n:translate="">Remarks</span>
  </h3>
  <div class="well">
  ...
  </div>
</div>
```
Then, it would look like this:

![imatge](https://user-images.githubusercontent.com/832627/43361157-314a6552-92c8-11e8-91f5-67af65ac2afb.png)